Deployment/README.md

```diff
@@ -47,6 +47,14 @@ The following are all valid invocations:
 You can find the output rap in the Deployment/Packages folder.
 *Note:* the output rap will override any other raps with the same version.
 
+## Creating a Nuget Package
```

Comment:
Can you please make the sections consistent?

  - [ ] Either change the above sections to match this one or change this one to match the above sections

Reply:
I don't understand what inconsistencies this is referring to.

Reply:
It looks like you picked up on it. It doesn't make much sense to have it repeated like this when you're writing it but it should be helpful to people reading the documentation. This way they don't have to look at the whole document and can just focus on what they care about.